phase1/tf.jsonnet

```diff
@@ -98,7 +98,7 @@
           ),
         },
       },
-      resource: {
+      resource+: {
```

Comment:
```
{
  field+: obj
}
```
is syntax sugar for
```
{
  field: super.field + obj
}
```

In the language spec, the form desugarring rule is:
```
desugarfield(fname +: expr, binds, b)	=	desugarfname(fname, b) : super.fname + local binds ; desugar(expr, true)
```

It basically says: when adding this object with another, instead of placing it's resource field with this field, add the two resource fields.